src/System.Runtime.Serialization.Xml/tests/SerializationTypes.cs

```diff
@@ -1165,4 +1165,24 @@ public override int GetHashCode()
         return this.DoubleProp.GetHashCode() ^ this.FloatProp.GetHashCode() ^
             this.DoubleField.GetHashCode() ^ this.SingleField.GetHashCode();
     }
+}
+
+public class TypeWithoutPublicSetter
+{
```

Comment:
Do we need to move the test case as well?

Reply:
The existing test contains many invalid type like invalidnestedtype, which will cause stackoverflow exception.